modules/apps/27-interchain-accounts/controller/ibc_module.go

```diff
@@ -66,16 +63,15 @@ func (im IBCModule) OnChanOpenTry(
 	version,
 	counterpartyVersion string,
 ) error {
-	return im.keeper.OnChanOpenTry(ctx, order, connectionHops, portID, channelID, chanCap, counterparty, version, counterpartyVersion)
+	return sdkerrors.Wrap(types.ErrInvalidChannelFlow, "channel handshake must be initiated by controller chain")
```

Comment:
Controller submodule returns errors for both `OnChanOpenTry` and `OnChanOpenConfirm` handshake callbacks